packages/react-ui-core/src/DatePicker/Calendar.js

```diff
@@ -67,9 +67,9 @@ export default class Calendar extends PureComponent {
     // NOTE: consitently create 5 rows so the calendar pads shorter months
     for (let i = 0; i <= 5; i += 1) {
       days.push(
-        <div key={`date-row-${i}`}>
+        <Fragment key={`date-row-${i}`}>
```

Comment:
Why did you change this to a fragment?

Reply:
How does this affect styling? It has no classname

Reply:
There's no dom element it's attached to. Did you check the output. It probably looks like this based on your change:
```
<div class="Calendar_DaysOfWeek">SuMoTuWeThFriSa</div>
```

Reply:
Sorry, I was thinking the wrong thing. But by removing this, it would cause there to be no rows which I assume you could do something with flex to address it, but it might be more difficult than just leaving the div.